### PR TITLE
Add some suggestions

### PR DESCRIPTION
**Pull request by jorgensd**
Some typos.

Add conda -> docker conversion

### CODE REVIEW

**Comment by jorgensd** [comment]
Fixes #2

**Comment by jorgensd** [comment]
@finsberg can I merge this?:D

File: docs/environment.md [hunk 1]
```diff
@@ -7,16 +7,54 @@ When reproducing the results of a research paper you would like to use the same
...
+channels:
+  - conda-forge
+dependencies:
+  - fenics-dolfin
+```
+
+We create the following `Dockerfile` to install the conda environment
+```docker
+# Use Docker image with preinstalled conda
+FROM conda/miniconda3
```
Comment (jorgensd) [inline docs/environment.md]:
```suggestion
FROM continuumio/miniconda3
```

**Review by finsberg — APPROVED** [review]

**Comment by finsberg** [comment]
> @finsberg can I merge this?:D

Go for it!

**Pull request merged**